seapy/model/grid.py

```diff
@@ -664,7 +664,7 @@ def ij(self, points):
         >>> idx = g.ij(a)
         """
 
-        from seapy.model.hindices import hindices
+        from seapy.external import hindices
```

Comment:
This is probably the most intrusive change.
The `Fortran` modules now live in a sub-module named `external`.
We can put them into the previous namespace,
but this is recommend so user know they are compiled from `Fortran` and not pure python.